test_local.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
+set -e
```

Comment:
The script terminates as soon as any line errors. Otherwise it's easy to miss if something failed if all the tests afterwards were successful, especially as their number grows.

Reply:
ok.  also, lets just make sure the other sets are put back in if they were commented out (I forget, but looked like it?)

Reply:
Oh yeah my bad, they were all passing at last push, I'll uncomment